how-to/workspace-platform-starter/client/src/framework/platform/platform-splash.ts

```diff
@@ -0,0 +1,138 @@
+import type OpenFin from "@openfin/core";
+import { getManifestCustomSettings } from "../settings";
+import { isStringValue } from "../utils";
+
+let win: OpenFin.Window | undefined;
+
+/**
+ * Open a splash screen.
+ */
+export async function open(): Promise<void> {
+	const app = await fin.Application.getCurrent();
+	const manifest = await app.getManifest();
+
+	const customSettings = await getManifestCustomSettings();
+
+	const disabled = customSettings?.splashScreenProvider?.disabled ?? false;
+
+	if (!disabled) {
+		win = await fin.Window.create({
+			name: "platform-splash",
+			uuid: "platform-splash",
```

Comment:
why does this have a custom uuid? This could issues with two platforms starting up at the same time. This would clash if two platforms were running? Maybe the progress window should be draggable so someone could move it if there were more than one platform starting up.

Reply:
Will switch to use platform id, not sure why I did that

Reply:
Done